test/sql/deparse-depesz/08-selects.d/21-in.psql

```diff
@@ -0,0 +1,6 @@
+--- TODO: Should we split the IN list items here?
```

Comment:
Something like
```sql
SELECT a
FROM b
WHERE
    c IN (1, 2, 3)
    AND d IN (
        'something longer', 'something longer 2', 'something longer 3',
        'something longer 4', 'something longer 5', ''
    )
```
? Yeah, that might be nice.

Reply:
Yup. I will however defer this for now, it seems, based on doing some more investigation just now.

Figuring out how to make this work is non-trivial, and I'd like to get a first version of this code committed soon (and I don't think fixing this will break the API).